tests/integration/test_workflow_changes_validation.py

```
@@ -0,0 +1,365 @@
+    """Test security best practices in modified workflows."""
+    
+    def test_workflows_use_pinned_action_versions(self):
+        """Verify workflows use specific action versions, not 'latest'."""
+        workflows_dir = Path(".github/workflows")
+        
+        for workflow_file in workflows_dir.glob("*.yml"):
+            with open(workflow_file, 'r') as f:
+                workflow = yaml.safe_load(f)
+            
+            # Check all jobs and steps
+            for job_name, job in workflow.get('jobs', {}).items():
```

Ruff (0.14.8)

Loop control variable `job_name` not used within loop body

Rename unused `job_name` to `_job_name`

(B007)